ntoskrnl/include/internal/ke_x.h

```diff
@@ -996,7 +996,7 @@ KiComputeDueTime(IN PKTIMER Timer,
 
 //
 // Called from Unlink and Queue Insert Code.
-// Also called by timer code when canceling an inserted timer.
+// Also called by timer code when cancelling an inserted timer.
```

Comment:
Unrelated changes outside translations (and above).

Reply:
It's not translation, it's previous  PR  to spell checking corrections.

Reply:
Fixed